dscptag.nft

```diff
@@ -20,7 +20,7 @@ define wan = "eth1.2" # change me
 
 table inet dscptag # forward declaration so the next command always works
 
-delete table inet dscptag # clear all the rules
```

Comment:
I think we probably want to use delete table rather than flush. This ensures everything is deleted. But we also still want the flags dynamic; for each set...

Reply:
nftables actually guarantees a reload is atomic as far as I know. Of course the tables would be empty yes, but it won't try to use sets that don't exist etc. Fast streams will hit these quickly so it'll be seconds at most that you'd have a burst of acks.

Reply:
Made the change and reverted to `delete table`